app/controllers/admin/api/services_controller.rb

```diff
@@ -42,11 +42,14 @@ def index
   ##~ op.parameters.add :name => "description", :description => "Description of the service to be created.", :dataType => "string", :required => false, :paramType => "query"
   ##~ op.parameters.add :name => "deployment_option", :description => "Deployment option for the gateway: 'hosted' for APIcast hosted, 'self_managed' for APIcast Self-managed option", :dataType => "string", :allowMultiple => false, :required => false, :paramType => "query"
   ##~ op.parameters.add :name => "backend_version", :description => "Authentication mode: '1' for API key, '2' for App Id / App Key, 'oidc' for OpenID Connect", :dataType => "string", :allowMultiple => false, :required => false, :paramType => "query"
+  ##~ op.parameters.add :name => "backend_api_id", :description => "ID of the backend api that will be associated with the service", :dataType => "string", :allowMultiple => false, :required => false, :paramType => "query"
```

Comment:
I think that we shouldn't document it until it is available for everyone 🤔 WDYT @hallelujah ?

Reply:
We should not you are right
All using API endpoints. There is a workflow that is broken, as it is with this removal:

1. Create a Service without `api_backend` parameter
2. `Proxy Config Promote
/admin/api/services/{service_id}/proxy/configs/{environment}/{version}/promote.json`
3. We should not be able to deploy as `api_backend` (private_endpoint) is blank

The customer should update either Service with api_backend or Proxy with private_endpoint or api_backend

This should be transitional state.

So this is broken for quick service creation, as it needs another UPDATE to deploy the proxy